### DISCUSSION TITLE
What is the status of [labs/gen-wrapper-angular] to generate angular bindings for web components?

### DISCUSSION THREAD

**Discussion by koenvd** [lit/lit#4525] [General]
I noticed that generating bindings seem to be supported for React & Vue.  

The status to generate Angular bindings is a bit unclear.  The repo contains code to generate them but I didn't find any documentation.  

Would help if anyone could give a hint about the status ...

**Comment by augustjk** [comment]
The Angular wrapper isn't actively being worked on right now. We never got to the point of publishing it or writing docs for it. Realistically I'm not sure when we'll be able to get back to it.

While I do think wrapper generation is important to really get web components to interop well, CLI wrapper generation is difficult to get write across many projects. It ends up being easier for project owners to roll their own code gen.

**Reply by gethari** [reply]
is there a documentation for `react/vue` ?

**Reply by augustjk** [reply]
I don't think we have docs for the wrapper generators outside of https://github.com/lit/lit/tree/main/packages/labs/cli
They're very experimental and only work for specific project setups.

The individual framework wrapper logic are in https://github.com/lit/lit/tree/main/packages/labs/gen-wrapper-react and https://github.com/lit/lit/tree/main/packages/labs/gen-wrapper-vue
Looking at e2e test goldens there might give an idea of what kind of wrappers they generate.

**Reply by gethari** [reply]
@augustjk thank you, when I try to run the cli, I am facing this issue https://github.com/lit/lit/issues/4078 which seems to be unresolved yet, if you knew any workaround please let me know
